js/index.js

```diff
@@ -239,8 +256,27 @@ onload = function() {
   Inspector.init();
   Inspector.showPane('pane-info');
 
+
+  // Handle file upload
+  $('#file_upload').onchange = async changeEv => {
+    changeEv.preventDefault();
+
+    const files = changeEv.target.files;
+    if (!files || files.length !== 1) {
+      return;
```

Comment:
No error message, because it might be the case that there is currently 1 file, and the user clicks again and cancels.

Reply:
This is the sort of thing that's good to have as a comment in the code.